dev-python/certifi/certifi-2020.12.5.recipe

```diff
@@ -3,11 +3,11 @@ DESCRIPTION="Certifi is a carefully curated collection of Root Certificates \
 for validating the trustworthiness of SSL certificates while verifying the \
 identity of TLS hosts. It has been extracted from the Requests project."
 HOMEPAGE="https://pypi.org/project/certifi/"
-COPYRIGHT="2011-2019 Kenneth Reitz"
+COPYRIGHT="2011-2020 Kenneth Reitz"
 LICENSE="MPL v2.0"
 REVISION="1"
 SOURCE_URI="https://files.pythonhosted.org/packages/62/85/7585750fd65599e88df0fed59c74f5075d4ea2fe611deceb95dd1c2fb25b/certifi-$portVersion.tar.gz"
-CHECKSUM_SHA256="e4f3620cfea4f83eedc95b24abd9cd56f3c4b146dd0177e83a21b4eb49e21e50"
+CHECKSUM_SHA256="1a4995114262bffbc2413b159f2a1a480c969de6e6eb13ee966d470af86af59c"
```

Comment:
Downloading failed, you probably need to adjust SOURCE_URI for this to work:
https://files.pythonhosted.org/packages/06/a9/cd1fd8ee13f73a4d4f491ee219deeeae20afefa914dfb4c130cfc9dc397a/certifi-2020.12.5.tar.gz